projects/aquadex/index.js

```diff
@@ -0,0 +1,3 @@
+const { uniTvlExport } = require('../helper/unknownTokens')
+
+module.exports = uniTvlExport('water', '0xc8F2534FF7c88EFeacF4Fdb0E81D87c6235C3bEA')
```

Comment:
is this a univ2 factory contract? any idea why we have an rpc failure here? which network explorer should we use to debug this?

Reply:
it's uniswap factory v3, hm, should I update to v2?

https://rpc.waterfall.network/
chainId: 181

Reply:
https://explorer.waterfall.network